solr/core/src/java/org/apache/solr/cli/AssertTool.java

```diff
@@ -131,7 +131,9 @@ public List<Option> getOptions() {
 
   @Override
   public int runTool(CommandLine cli) throws Exception {
-    verbose = cli.hasOption(SolrCLI.OPTION_VERBOSE.getOpt());
+    debug =
+        cli.hasOption((SolrCLI.OPTION_DEBUG.getOpt()))
+            || cli.hasOption(SolrCLI.OPTION_VERBOSE.getOpt());
```

Comment:
With an `OptionGroup` this could be simplified to `debug = cli.hasOption(SolrCLI.OPTION_DEBUG)`, where `SolrCLI.OPTION_DEBUG` is the `OptionGroup` that includes verbose and debug options.

Note that the help outputs may be different if using option groups.

Reply:
I think I need to dig more in and write a unit test to prove that works!